README.md

```diff
@@ -91,7 +91,7 @@ Setting up and using the API client is simple, just pass in your authenticator o
 
 # Pass the authenticator into the VpcV1 service
 vpc_v1 = IbmVpc::VpcV1.new(
-  version: "2021-08-03" # Will default to the latest version if not specified
+  version: "2022-07-12" # Will default to the latest version if not specified
```

Comment:
Lets keep the API version.

Reply:
And please avoid force push .. If possible create a new PR to trigger semantic release bot correctly.